mix.exs

```diff
@@ -46,14 +59,35 @@ defmodule SaladUI.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:phoenix_live_view, "~> 0.20.1"},
       {:tails, "~> 0.1.5"},
+      {:phoenix, "~> 1.7"},
+      {:heroicons, "~> 0.5.3"},
+      {:phoenix_ecto, "~> 4.4"},
+      {:phoenix_html, "~> 4.0"},
```

Comment:
I don't think we need this much dependencies

Reply:
We don't need ecto for now, but form inputs tests might need it if we want to test them with Phoenix.